packages/venia-concept/src/components/Head/headProvider.js

```diff
@@ -0,0 +1,19 @@
+import React, { useEffect } from 'react';
+import { HeadProvider as _HeadProvider } from 'react-head';
```

Comment:
I'm not a huge fan of the underscore as the first character of a React component 😬

Reply:
What shall I change it to? I wanted to signify vanila `HeadProvider` as` _HeadProvider`.

Reply:
Interesting. Let me get this done.

Reply:
@supernova-at The pattern of naming things `VeniaHeadProvider` after the package is kind of frustrating in practice. These components get used in implementations that aren't identified as Venia anymore and that naming doesn't feel good to see, so we rewrite and maintain that rewrite. It makes upstream integration more complex.